modules/nf-core/qualimap/bamqc/tests/main.nf.test

```diff
@@ -9,11 +9,35 @@ nextflow_process {
     tag "qualimap/bamqc"
 
     test("homo_sapiens [bam]") {
-
         when {
-            params {
-                outdir   = "$outputDir"
+            process {
+                """
+                gff = []
+
+                input[0] = Channel.of([
+                    [ id:'test', single_end:false ], // meta map
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/bam/test.paired_end.sorted.bam', checkIfExists: true)
+                ])
+                input[1] = gff
+                """
             }
+        }
+
+        then {
+            assertAll (
+                { assert process.success },
+                { assert snapshot(
+                    file("${process.out.results[0][1]}/qualimapReport.html").name,
+                    path("${process.out.results[0][1]}/genome_results.txt"),
```

Comment:
why is one of these file, the other path?

Reply:
The first one is just snapshoting the file name, the second the file content